code/modules/mob/living/carbon/carbon.dm

```diff
@@ -89,6 +89,8 @@
 
 
 /mob/living/carbon/electrocute_act(shock_damage, obj/source, siemens_coeff = 1, override = 0, tesla_shock = 0)
+	if(dna && dna.check_mutation(SMES_HUMAN))

```

Comment:
Technically we dont have a clear way of asserting should species things override mutations or should mutations override species things
THus any behaviour is for now acceptable